network/protocol.go

```diff
@@ -107,6 +110,17 @@ func isFullCapability(c *capability.Capability) bool {
 	return fullCapability.IsSameAs(c)
 }
 
+// temporary convenience functions for "bootnode"
+func newBootnodeCapability() *capability.Capability {
+	c := capability.NewCapability(CapabilityID, 16)
+	c.Set(capabilitiesBootnode)
+	return c
+}
+
+func isBootnodeCapability(c *capability.Capability) bool {
```

Comment:
Is this integrate and used?

Reply:
I'd also like to see it be actually used somewhere :)

Reply:
i thought you're gonna bark at me for not adding this in a separate PR @nolash